AntennaTracker/Parameters.cpp

```diff
@@ -416,6 +431,21 @@ const AP_Param::Info Tracker::var_info[] = {
     // @Increment: 1
     // @Units: Hz
     // @User: Standard
+
+    // @Param: YAW2SRV_SMAX
+    // @DisplayName: Yaw slew rate limit
```

Comment:
Is it "Yaw Rate Slew Limit" perhaps?

Reply:
@rmackay9 I don't know exactly how this works but if it is limiting the rate of the command out to the actuator, then it is a slew rate limit.  It is not limiting yaw rate.